pass-journal-loader/pass-journal-loader-nih/src/main/java/org/eclipse/pass/loader/journal/nih/JournalFinder.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2017 Johns Hopkins University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.eclipse.pass.loader.journal.nih;
+
+import java.util.List;
+
+import org.eclipse.pass.support.client.model.Journal;
+
+/**
+ * @author apb@jhu.edu
+ */
+public interface JournalFinder {
+    /**
+     * Find the journal best matching the provided data.
+     * If the journal already exists, return SKIP
+     *
+     * @param nlmta
```

Comment:
I noticed there are some javadoc warning when running `verify`

Reply:
Fixed. I also found that JournalRecord is not actually used.